react/todo-app-fe/src/redux/sagas/deleteTodoSaga.js

```diff
@@ -0,0 +1,26 @@
+import {call, put, takeEvery} from 'redux-saga/effects';
+import { deleteTodoSuccess, deleteTodoFail, getUserTodos } from '../actions/todos';
+import axiosInstance from '../../axios';
+
+function removeTodo(todo) {
```

Comment:
Ja bih posle na sledecem pravio servise za ovo, ne bih delio todoSage na vise njih nego tipa folder:
- todo:
    - reducers.js
    - sagas.js

- user:
    - reducers.js
....

Ali ok ne moras sada menjati, ali bih pogledao kako radimo https://github.com/Vivify-Ideas/expo-boilerplate/tree/master/services
Ovde za HttpServic, ApiService i kako ostali nasledjuju to